File: remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    
    let i = 0;
    
    for(j=0; j<nums.length; j++){
        if(nums[j] !== nums[i]){
            nums[i+1]  = nums[j];
            i += 1;
        }
    }
    
    return i+1;
    
};

File: remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        i = 0
        for j in range(0, len(nums)):
            if nums[i] != nums[j]:
                nums[i+1] = nums[j]
                i+=1
        
        return i+1
        
        
        